src/relay/pass/type_infer.cc

```diff
@@ -163,7 +174,19 @@ class TypeInferencer : private ExprFunctor<Type(const Expr&)> {
     return ret;
   }
 
-  // Visitor logics
+  [[noreturn]] void ReportFatalError(const Expr& expr, const std::stringstream& err) {
```

Comment:
I think we probably don't need this function because there is a constructor for `Error(const std:: stringstream& err)`, is it correct?

Reply:
Yes this was left over from my trying to debug why the implicit constructor wasn't firing on the old version of gcc. I found out it was because `<<` has a different return type on my MacBook vs. on the build machine.

Reply:
@jroesch yeah, `<<` actually returns `ostream`, that's probably why you created `StringStream`